packages/http-client-java/emitter/src/code-model-builder.ts

```diff
@@ -2584,9 +2579,7 @@ export class CodeModelBuilder {
   }
 
   private logWarning(msg: string) {
-    if (this.loggingEnabled) {
```

Comment:
I removed this part, as it seems not right to have the `loggingEnabled` controlling the warning.